### PR TITLE
doc: add 12.2.2 release notes

### PR DESCRIPTION
**Pull request by theanalyst**
Also clean up PendingReleaseNotes & update release table

### CODE REVIEW

File: doc/release-notes.rst [hunk 1]
```diff
@@ -2,6 +2,127 @@
  Release Notes
 ===============
 
+v12.2.2 Luminous
+================
+
+This is the second bugfix release of Luminous v12.2.x long term stable
+release series. It contains a range of bug fixes and a few features
+across CephFS, RBD & RGW. We recommend all the users of 12.2.x series
```
Comment (liewegas) [inline doc/release-notes.rst]: and bluestore

Comment (theanalyst) [reply doc/release-notes.rst]: ack

File: doc/release-notes.rst [hunk 2]
```diff
@@ -2,6 +2,127 @@
...
+update.
+
```
Comment (liewegas) [inline doc/release-notes.rst]: There are also several major items backported to this release:

- Standby ceph-mgr daemons now redirect requests to the active messenger, easing configuration of tools and humans that access the web dashboard, restful API, or other ceph-mgr module services.
- The prometheus module has several significant updates and improvements.
- The new balancer module enables automatic optimization of CRUSH weights to balance data across the cluster.
- The ceph-volume tool has been updated to include support for BlueStore as well as FileStore.  The only major missing ceph-volume feature is dm-crypt support.

Comment (theanalyst) [reply doc/release-notes.rst]: thanks

File: doc/release-notes.rst [hunk 3]
```diff
@@ -2,6 +2,135 @@
...
+series. It contains a range of bug fixes and a few features across Bluestore,
+CephFS, RBD & RGW. We recommend all the users of 12.2.x series update.
+
+For more detailed information, see :download:`the complete changelog <changelog/v12.2.2.txt>`.
+
+
+Notable Changes
+---------------
+
+* Standby ceph-mgr daemons now redirect requests to the active messenger, easing configuration of tools and humans that access the web dashboard, restful API, or other ceph-mgr module services.
```
Comment (smithfarm) [inline doc/release-notes.rst]: Two things with this bullet point:

1. the line is very long - can it be wrapped/indented to fit into 80 characters like the lines in the intro above?
2. the formulation "easing configuration of tools and humans . . ." strikes me as odd? Maybe just drop "and humans"?

File: doc/release-notes.rst [hunk 4]
```diff
@@ -2,6 +2,135 @@
...
+* The prometheus module has several significant updates and improvements.
+* The new balancer module enables automatic optimization of CRUSH weights to balance data across the cluster.
+* The ceph-volume tool has been updated to include support for BlueStore as well as FileStore. The only major missing ceph-volume feature is dm-crypt support.
+* RGW's dynamic bucket index resharding is disabled in multisite environments.
```
Comment (smithfarm) [inline doc/release-notes.rst]: Here, as a reader, I'm missing an explanation of why it was disabled?

Comment (theanalyst) [reply doc/release-notes.rst]: addressed

File: doc/release-notes.rst [hunk 5]
```diff
@@ -2,6 +2,141 @@
...
+  configuration for tools & users accessing the web dashboard, restful API, or
+  other ceph-mgr module services.
+* The prometheus module has several significant updates and improvements.
+* The new balancer module enables automatic optimization of CRUSH weights to
+  balance data across the cluster.
+* The ceph-volume tool has been updated to include support for BlueStore as well
+  as FileStore. The only major missing ceph-volume feature is dm-crypt support.
+* RGW's dynamic bucket index resharding is disabled in multisite environments,
+  as it can cause inconsistencies in replication of bucket indexes to remote
+  sites
```
Comment (smithfarm) [inline doc/release-notes.rst]: full stop after "sites"

**Comment by theanalyst** [comment]
jenkins test docs

**Comment by theanalyst** [comment]
jenkins render docs

**Pull request merged**